src/lib/helpers.ts

```diff
@@ -1,7 +1,21 @@
-import fetch from 'node-fetch';
+import * as fetch from 'isomorphic-fetch';
 
-export async function request(uri: string, headers: { [header: string]: string; } = {}) {
-    const req = await fetch(uri, { compress: true, timeout: 30000, headers });
+/**
+ * Make a http(s) request to a json api.
+ *
+ * Rejects when a network error occured.
+ * 4xx and 5xx response codes are not network errors, and will resolve the promise.
+ *
+ * @param uri The uri to request the data from.
+ * @param headers Headers to attach to the request.
+ * @return The body of the response, parsed as json.
+ * It returns with `null` when the api does not respond with `200 OK`.
+ */
+export async function request(
```

Comment:
This does not need the extra return/formatting. Is this is preferred; then a separate PR should be made converting the entire code base to the same format.